gcb/release/cloudbuild.yaml

```diff
@@ -47,7 +47,7 @@ steps:
   - "./anago"
   - "${_RELEASE_BRANCH}"
   - "${_NOMOCK}"
-  - "${_BUILDVERSION}"
+  - "--buildversion=${_BUILDVERSION}"
```

Comment:
nitpick: could we standardize on `--build-version` or `--buildversion` (in what the user passes to the command and what gets passed to cloudbuild)? I think it has been a source of confusion in the past.

Reply:
I'd rather not change the flag in anago since we will not use anago any more in the mid term. This means we also will remove the cloudbuild.yaml files as a cleanup step when anago goes away.

Reply:
@hasheddan -- Yeah, I've gone back and forth on this. anago uses `--buildversion` and krel uses `--build-version`.

My preference is the latter (kebab-case).
I previously I've been pretty risk-averse about changing anago flags, but it's probably a lot safer to change it today.

Then again, we're on the cusp of destroying anago altogether, so maybe the potential headache isn't even worth it?